main/codesig/src/JvmModel.scala

```diff
@@ -223,7 +223,10 @@ object JvmModel {
         args.addOne(JType.read(s.substring(index, split + 1)))
         index = split + 1
       }
-      new Desc(args.result(), JType.read(s.substring(closeParenIndex + 1)))
+      new Desc(
+        collection.immutable.ArraySeq.unsafeWrapArray(args.result()),
```

Comment:
If this is `scala.collection.immutable.ArraySeq`, then I think it's reasonable to just import it at the top of the file.